qiskit/pulse/reschedule.py

```diff
@@ -12,190 +12,10 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Basic rescheduling functions which take a Schedule (and possibly some arguments) and return
-a new Schedule.
-"""
+"""Deprecated transforms. Moved to "qiskit.pulse.transforms"."""
```

Comment:
This should be discussed -- this change is also out of scope for the pulse builder interface

Reply:
Moved in a separate PR #4251